src/modules/auth/auth_repository.ts

```diff
@@ -54,8 +54,25 @@ export namespace Auth {
     })
   }
 
-  export const findByEmail = async (requestBody: Entity.RequestBodySignIn) => {
-    return Users().where('email', requestBody.email).first()
+  export const findByEmail = async (requestBody: Entity.FindByEmail) => {
+    return Users()
+      .where('email', requestBody.email)
+      .first()
+  }
+
+  export const findByUserId = async (id: string) => {
+    return Users()
+      .where('id', id)
+      .first()
+  }
+
+  export const findByRefreshToken = async (requestBody: Entity.RequestBodyRefreshToken) => {
+    return OauthTokens()
+      .select('users.*')
```

Comment:
don't use select * please use select specific field

Reply:
yes i forgot to select specific field, thank's

Reply:
yes because YAGNI is our. :cupid:

Reply:
@rachadiannovansyah bebebnya hadi